search/docker-compose.yml

```diff
@@ -13,3 +13,4 @@ services:
       - "logger.level=DEBUG"
       - "discovery.type=single-node"
       - "xpack.security.enabled=false"
+      - "cluster.routing.allocation.disk.threshold_enabled=false"
```

Comment:
What does this do?

Reply:
When testing, I found that ES would often get into a red state because it passed the high watermark for disk space in the docker container.  This switches off that check so that it will run even if the "disk" is 95% full.